logstash-output-rollbar.gemspec

```diff
@@ -1,7 +1,7 @@
 Gem::Specification.new do |s|
 
   s.name            = 'logstash-output-rollbar'
-  s.version         = '0.1.2'
+  s.version         = '0.1.3'
```

Comment:
Good point. Updated to 0.2.0 in https://github.com/rollbar/logstash-output-rollbar/commit/8bf3b32217ba3ec968e4000e0254aa494286c567